float32 distance_wall_left
float32 distance_wall_right
float32 distance_wall_front
---
float32[] obstacle_x
float32[] obstacle_y